scripts/chromium.pref

```diff
@@ -0,0 +1,23 @@
+# Note: 2 blank lines are required between entries
```

Comment:
The rest of the file suggests otherwise 😅 (I think it's _at least one_ blank line, but tbh there's probably no need for the comment.)
```suggestion
```

Reply:
Or if you prefer, could just link to the manpage.

```
# https://manpages.ubuntu.com/manpages/noble/man5/apt_preferences.5.html
```